### ISSUE TITLE
Incorrect login appears to authenticate

### ISSUE THREAD

**Issue by timmmmyboy** [elmsln/HAXcms#571]
Checked in a few different browsers. Environment is PHP 7.4.7 on Apache 2.4.43. A brand new install, pass the user and password to install.php. Login with that is fine. However it appears any login actual works and exposes the UI (actual functions fail to work with console errors, but I'm assuming the no one should even get past the login screen much less see the site content.

**Comment by btopro** [comment]
brand new install with the latest version or the latest version on Reclaim? Not able to replicate locally but we're on 1.2.8

**Comment by timmmmyboy** [comment]
This was 1.2.0 on Reclaim Cloud (built an installer for it, in Marketplace under Education if you want to take a peek). I can try 1.2.8 and see if that changes things. You're bumping releases like every day, lol.

**Comment by btopro** [comment]
yeah, I was yelled at to bump versions bc we have a container rebuild on tagged release

**Comment by timmmmyboy** [comment]
Confirmed even with 1.2.8 same thing is happening, clean browser session, I can put anything in the login form and it authenticates.

<img width="1207" alt="Screen Shot 2020-07-19 at 2 22 16 PM" src="https://user-images.githubusercontent.com/3890972/87882039-55029c80-c9cb-11ea-8b0a-86aee6caeeb0.png">

<img width="1207" alt="Screen Shot 2020-07-19 at 2 22 23 PM" src="https://user-images.githubusercontent.com/3890972/87882042-57fd8d00-c9cb-11ea-85b6-3018efbd9158.png">

Let me know what else might help for diagnosing. The install is in the Marketplace so you can reproduce in Reclaim Cloud but I don't want to publicize until the bug is patched.

**Comment by btopro** [comment]
I'll have to fire it up there and see what's up because I don't have this happen when running it locally. It's "sorta" a security issue though only sorta since as you mentioned everything fails to actually function but still definitely worth fixing cause.... it shouldn't do that :p

**Comment by btopro** [comment]
finally able to replicate this and I see the issue; doing some work locally now to see if I can fix it for the next release

**Comment by btopro** [comment]
This should be fixed in 1.2.11. Was related to PHP version + `http_response_code` not being used. Should be good now.
